modules/gathering-crio-logs.adoc

```diff
@@ -22,7 +22,7 @@ endif::openshift-rosa,openshift-dedicated[]
 
 .Procedure
 
-. Gather CRI-O journald unit logs. The following example collects logs from all control plane nodes (within the cluster:
+. Gather CRI-O journald unit logs. The following example collects logs from all control plane nodes (within the cluster):
```

Comment:
I might remove the parenthesis. It's not needed and we try to avoid parenthesis. Thanks for catching this. 
```suggestion
. Gather CRI-O journald unit logs. The following example collects logs from all control plane nodes (within the cluster):
```